tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -355,6 +355,34 @@ public function getFolderId(string $user, string $spaceName, string $folderName)
 		return $responseArray[0][0];
 	}
 
+	/**
+	 * @param string $user
+	 * @param string $spaceName
+	 *
+	 * @return mixed
+	 * @throws GuzzleException
+	 * @throws JsonException
+	 */
+	public function getPrivateLink(string $user, string $spaceName): string {
+		$this->setSpaceIDByName($user, $spaceName);
```

Comment:
the task of this function should be only to return the private link
```suggestion
```

Reply:
This is actually to set the `$SPACE_ID_FROM_OCIS` variable in core
https://github.com/owncloud/ocis/blob/1f4780d730d5a55ab56c42baaf2eba2eec3e2641/tests/acceptance/features/bootstrap/SpacesContext.php#L261-L265

so that we can send requests to space through the core's methods

Reply:
ohh! decision is yours :+1: